### PR TITLE
(#1809) Float Util Code Examples

### LINKED ISSUE #1809: Float Utility Page Example Code needs to be updated and remove presentation classes
### Issue description
Float Utility Page Example Code needs to be updated and remove presentation classes from the DOM examples. Created Demonstration class for Float Items.

> **ESTIMATE** TBD

### Steps to reproduce the issue

1.  go to https://designsystem-dev.cancer.gov/develop/foundations/layout-positioning/float
2.  See code examples - they have extra site css that is not required

### What's the expected result?
- To see only the dom and css classes needed, no extra presentation classes.

### What's the actual result?
- extra classes in dom

### Additional details / screenshot
note from Layout/Flex PR
https://github.com/NCIOCPL/ncids/pull/1794

### Related Tickets
* Issue #9999
* Issue #9999

### PR DESCRIPTION
**Pull request by karlikpj**
Closes #1809

## Pull Request Details

_Add description_

Closes #<TICKET-NUMBER>

## Author PR Checklist
Items that the author of the PR is responsible for checking before submitted the PR.

### General:
- [x] I have reviewed the acceptance criteria defined in the ticket and ensured the work has been completed.
- [x] The commit message passes [all quality commit message standards](https://github.com/NCIOCPL/ncids/wiki/Creating-Quality-Commit-Messages).
- [x] Unit tests have been updated or created to reflect any javascript changes.
- [x] Storybook scenarios have been updated or created to reflect any html/css/js changes.

### Accessibility:
- [x] WCAG 2.1 Level AA requirements have been met.

### Development:
- [x] Any new or updated javascript code has 100% unit test coverage.
- [x] New or updated breakpoints have regression images.
- [x] Breaking changes have been thoroughly documented in the PR.

## Product Reviewer PR Checklist
Items the product team is responsible for reviewing.

### General:
- [x] There are no unexpected or unapproved regression image changes. 
- [x] Functionality of interactive elements meet the acceptance criteria.
- [x] The product is visually and functionally the same across the different browsers.

### Accessibility:
- [x] AxeDev Tools: there are no new or outstanding accessibility issues introduced in this PR.
- [x] Lighthouse: scores have not noticeably decreased during this PR.
- [x] Wave: there are no new errors or contrast errors introduced in this PR.

## Design Reviewer PR Checklist
Items the design team is responsible for reviewing.  

### General:
- [x] New or updated features introduced in this PR are developed mobile-first.
- [x] Breakpoint changes and regression images match those breakpoints.
- [x] This PR has been tested in all supported browsers at all breakpoints.

## Developer Reviewer PR Checklist
Items the development team is responsible for reviewing.

### General:
- [ ] New code passes code quality standards set by industry standards.
- [ ] The expected Storybook stories have been added or updated for the new or updated feature.
- [ ] The expected unit tests have been added or updated for the new or updated feature.

### Accessibility:
- [ ] VoiceOver: Described content matches with what was expected.
- [ ] Keyboard navigation: new or updated features and content are navigable via the keyboard.

### CODE REVIEW

File: pnpm-lock.yaml
Comment (olitharp-nci) [inline pnpm-lock.yaml:1]: you shouldn't have a pnpm-lock change. revert this, make sure you have both node 20 and pnpm 9 installed

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

File: docs/content/foundations/layout-positioning/float.mdx [hunk 1]
```diff
@@ -10,47 +10,53 @@ utility_modules:
...
+        - class_name: 'float-left'
+        - class_name: 'float-right'
     utility_examples:
       - heading:
         description: |
-        preview: true
+        preview: false
         code: |
-          <section className="utility-examples">
+          <section>
```
Comment (olitharp-nci) [inline docs/content/foundations/layout-positioning/float.mdx:25]: remove section element wrappers

File: docs/content/foundations/layout-positioning/float.mdx [hunk 2]
```diff
@@ -10,47 +10,53 @@ utility_modules:
...
-      classes: []
+      type: 'clearfix'
+      classes:
+        - class_name: 'clearfix'
     utility_examples:
       - heading: Example usage
-        preview: true
+        preview: false
         code: |
           <section className="utility-examples">
```
Comment (olitharp-nci) [inline docs/content/foundations/layout-positioning/float.mdx:54]: shouldn't be here either

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

**Comment by olitharp-nci** [comment]
Don't resolve conversations, let the PR reviewer do it.

**Review by olitharp-nci — APPROVED** [review]

**Comment by sarinapadilla** [comment]
@karlikpj I noticed that the "Component Preview" headings around the Example Usage sections are missing when I compare the [PR page](https://designsystem-dev.cancer.gov/pr-1811/foundations/layout-positioning/float) to the [develop page](https://designsystem-dev.cancer.gov/develop/foundations/layout-positioning/float).

@adrianacastaneda @blilianyu @monika-jaeger what are your thoughts on this change?

**Review by sarinapadilla — APPROVED** [review]
Per @karlikpj, the removal of the "Component Preview" is expected.
> Its a different use of the component preview. The one on dev is Example Usage header - code preview then code snippet. But in the PR version it's Generated Preview - Example Usage Header - code snippet - we're omitting the code preview and using a generated example as the example uses extra css we don't want users to copy but needed for presentation. But that header is a part of the code preview/snippet module and can't be moved.

So this looks good to me!

**Comment by olitharp-nci** [comment]
.

**Comment by olitharp-nci** [comment]
.

**Comment by olitharp-nci** [comment]
.

**Pull request merged**